specification/deploymentmanager/resource-manager/readme.md

```diff
@@ -0,0 +1,71 @@
+# Azure Deployment Manager
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for Deployment Manager.
+
+
+---
+## Getting Started
+To build the SDK for Deployment Manager, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
+
+> `autorest`
+
+To see additional help and options, run:
+
+> `autorest --help`
+---
+
+## Configuration
+
+
+### Basic Information
+These are the global settings for the DeploymentManager API.
+
+``` yaml
+openapi-type: arm
+tag: package-2018-09-01-preview
+```
+
+### Suppression
+
+``` yaml
+directive:
+  - suppress: EnumInsteadOfBoolean
+    reason:  The boolean properties in the specification are actually boolean values in the Deployment Manager application model.
+    from: deploymentmanager.json
+  - suppress: TrackedResourceListByImmediateParent
+    reason: Not available during preview.
+    from: deploymentmanager.json    
+```
+
+### Tag: package-2018-09-01-preview
+
+These settings apply only when `package-2018-09-01-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-2018-09-01-preview'
+input-file:
+- Microsoft.DeploymentManager/preview/2018-09-01-preview/deploymentmanager.json
+```
+
+---
+# Code Generation
+
+
+## Swagger to SDK
+This section describes what SDK should be generated by the automatic system.
+This is not used by Autorest itself.
+
+## C#

```

Comment:
please add all the languages that your SDK should eventually be published in

Reply:
@hovsepm synced with @salameer on excluding this from auto generating SDKs. Is it still required to have the required declarations in this file?

Reply:
@deveshguha it is not required to generate private SDK but is required for this PR to be merged eventually.

Reply:
Got it. Updated the file referencing other services to add sdks and relevant settings.